.config/nextest.toml

```diff
@@ -1,11 +1,19 @@
+[test-groups]
+against-tilt = { max-threads = 1 }
```

Comment:
this causes tests starting with `test_k8s_integration_` to be run in serial in CI